qa/integration-steps/src/main/java/org/eclipse/kapua/qa/integration/steps/DockerSteps.java

```diff
@@ -31,12 +31,21 @@
 import cucumber.runtime.java.guice.ScenarioScoped;
 import org.apache.activemq.command.BrokerInfo;
 import org.eclipse.kapua.qa.common.StepData;
+import org.apache.shiro.SecurityUtils;
```

Comment:
Please change header to: " * Copyright (c) 2019, 2020 Eurotech and/or its affiliates and others"